### DISCUSSION TITLE
Is there a feature in Patrol similar to 'skipOffstage'?

### DISCUSSION THREAD

**Discussion by haomun0731** [leancodepl/patrol#1682] [Q&A] [answered]
With Flutter tests, we can perform an action like this:

`expect(find.byType(LoginButton, skipOffstage: false), findsOneWidget);`

I'm curious if Patrol offers a similar capability. At the moment, I've implemented an alternative approach like this:

`await $.tester.drag(
    find.byType(Scaffold),
    const Offset(0, -200),
);`
`expect($(LoginButton), findsOneWidget);`

**Comment by bartekpacia** [comment, marked as answer]
No, we don't support it.

**Reply by haomun0731** [reply]
Thank you for clarifying.
